hammer/library/aws/ecs.py

```diff
@@ -198,7 +198,7 @@ def check(self, task_definitions=None):
                         logging.error(f"Access denied in {self.account} "
                                       f"(ecs:{err.operation_name})")
                     else:
-                        logging.exception(f"Failed to describe task definitions in {self.account} "
```

Comment:
Do we need to add "AccessDeniedException" error code into ClientError exception as was made to elasticsearch.py module?